info.txt

```diff
@@ -0,0 +1 @@
+I am louis, my second name is James Bond, my phone number is 07816093423
```

Comment:
Two comments here:
- can you rename the file to `text.txt` (you will need to rename the file locally, and run `git add text.txt`)
- can you add more personal information to the text? like an physical address, an email address and a url?